pkg/retry/retry.go

```diff
@@ -37,41 +37,65 @@ const (
 // events.
 func (s *Server) handleRetry() http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		now := time.Now().UTC().Format(time.UnixDate)
```

Comment:
Sorry, nit: I'd make this `time.Now().UTC()` so you have a real time object that you can use throughout.

Then when you need to put it in the query, call `.Format(time.UnixDate)`.